metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DepublishRecordIdService.java

```diff
@@ -239,12 +238,32 @@ public boolean canTriggerDepublication(MetisUser metisUser, String datasetId)
     } else {
       // If a (re-)index took place recently, or the status is not published, we can't depublish.
       final DatasetExecutionInformation executionInformation = orchestratorService
-              .getDatasetExecutionInformation(datasetId);
+          .getDatasetExecutionInformation(datasetId);
       result = executionInformation.getPublicationStatus() == PublicationStatus.PUBLISHED &&
-              executionInformation.isLastPublishedRecordsReadyForViewing();
+          executionInformation.isLastPublishedRecordsReadyForViewing();
     }
 
     // Done
     return result;
   }
+
+  Set<String> checkAndNormalizeRecordIds(String datasetId,
+      String recordIdsInSeparateLines) throws BadContentException {
+    return DepublishRecordIdUtils
+        .checkAndNormalizeRecordIds(datasetId, recordIdsInSeparateLines);
+  }
+
+  ResponseListWrapper<DepublishRecordIdView> createResponseListWrapper(List<DepublishRecordIdView> records, int page){
+    final ResponseListWrapper<DepublishRecordIdView> result = new ResponseListWrapper<>();
+    result.setResultsAndLastPage(records, depublishRecordIdDao.getPageSize(), page);
+    return result;
+  }
+
+  Workflow createNewWorkflow(){
+    return new Workflow();
+  }
+
+  DepublishPluginMetadata createNewDepublishPluginMetadata(){
+    return new DepublishPluginMetadata();
+  }
```

Comment:
Same as what I said before, may intention was to mock these objects and `verify` if specific method from them were called. Any suggestion as to what I could do otherwise?

Reply:
Solved